tests/env/components/echo_http_server.go

```diff
@@ -27,7 +27,7 @@ type EchoHTTPServer struct {
 	*Cmd
 }
 
-func NewEchoHTTPServer(port uint16, enableHttps bool, enableRootPathHandler, useWrongCert bool, mtlsCertFile string, disableHttp2 bool) (*EchoHTTPServer, error) {
+func NewEchoHTTPServer(port uint16, enableHttps bool, enableRootPathHandler, useWrongCert bool, mtlsCertFile string, disableHttp2, backendAlwaysRespondRST bool, backendRejectRequestNum, backendRejectRequestStatus int) (*EchoHTTPServer, error) {
```

Comment:
too many parameters,   can we use a struct for passing them?  it will be more reabable

Reply:
Done.